src/main/scala/edu/twitter/SentimentAnalyzer.scala

```diff
@@ -33,7 +33,7 @@ object SentimentAnalyzer extends App {
   modelService.start()
 
   val classifier = new Classifier(ssc)
-  val classifiedStream = classifier.createClassifiedStream()
+  val classifiedStream = classifier.createClassifiedStream(GradientBoostingModel.name)
   classifiedStream.foreachRDD(EsSpark.saveToEs(_, "twitter/sentiment"))
```

Comment:
Shouldn't we put different data from different models in different indices, or we can add a label called model to the data?

Reply:
Good point, will persist to different indices